pkg/reconciler/apis/apiexport/apiexport_reconcile.go

```diff
@@ -95,6 +95,18 @@ func (c *controller) reconcile(ctx context.Context, apiExport *apisv1alpha1.APIE
 		)
 	}
 
+	// check if any APIBindings are bound to this APIExport. If so, add a virtualworkspaceURL
+	apiBindings, err := c.getAPIBindingsForIdentityHash(apiExport.Status.IdentityHash)
```

Comment:
Would it be sufficient to match on the logical cluster + name of the APIExport (the reference from the APIBinding spec)?

Reply:
I was thinking that the identity hash would be more unique, but you're right since you can't have duplicate names.

I'll correct that, I think we already have an indexer for it, just need to switch to it.